filters/time.js

```diff
@@ -0,0 +1,25 @@
+(function () {
```

Comment:
wouldn't it easier to apply the predefined date filter? @CodingCarlos @javierLiarte

Reply:
@DevStarlight As far as I know, there's no default time filter, but just the [date filter](https://docs.angularjs.org/api/ng/filter/date), and it does not convert seconds into `mm:ss.sss` format, but shows in that format a date's moment ;) 

This makes me think in adding jsdocs to the filters, and maybe to all the code.